app/controllers/api/v1/reviews_controller.rb

```diff
@@ -1,21 +1,19 @@
 class Api::V1::ReviewsController < ApplicationController
+  before_action :authorize_user, except: [:index]
   skip_before_action :verify_authenticity_token, only: [:create, :destroy]
+
   def index
     city_id = params[:city_id]
     city_reviews = Review.where(city_id: city_id)
     render json: city_reviews
   end
 
   def create
-    if current_user != nil
-      review = Review.new(review_params)
-      if review.save
-        render json: review
-      else
-        render json: { error: review.errors.full_messages }, status: :unprocessable_entity
-      end
+    review = Review.new(review_params)
+    if review.save
+      render json: review
     else
-      render json: { errors: "Access Denied" }, status: 401
```

Comment:
this was correct. To Nick's point, what happens in the application when this `401` status code is received - your fetch invocation will have to be modified to handle for this response.